apollo-router/src/plugins/telemetry/config.rs

```diff
@@ -514,13 +531,26 @@ impl From<AttributeValue> for opentelemetry::Value {
         match value {
             AttributeValue::Bool(v) => Value::Bool(v),
             AttributeValue::I64(v) => Value::I64(v),
+            AttributeValue::U128(v) => Value::I64(v as i64),
```

Comment:
This is odd 🤔

Reply:
Ah yes, otel Value doesn't support u128 unfortunately